doc/Overview.md

```diff
@@ -19,37 +19,153 @@
 // <http://www.gnu.org/licenses/>.
 -->
 
-## Abstract
+\section OverviewIntro What is Pinocchio?
 
-We introduce Pinocchio, an open-source software framework that implements rigid body dynamics algorithms and their
-analytical derivatives. Pinocchio does not only include standard algorithms employed in robotics (e.g. forward and
-inverse dynamics) but provides additional features essential for the control, the planning and the simulation of
-robots. In this paper, we describe these features and detail the programming patterns and design which make Pinocchio
-efficient. We also offer a short tutorial for easy handling of the framework.
+Pinocchio is a library for efficiently computing the dynamics (and derivatives) of a robot model, and behing, any articulated rigid-body model you would like (avatar in a simulator, skeletal model for biomechanics).
+Pinocchio is one of the most efficient library for computing the dynamics of articulated bodies.
+It implements the classical algorithms following the methods described in Featherstone 2009 book (many thanks to him).
+It also introduce efficient variations of some of these algorithms, some new ones and algorithms to compute the derivatives of the main algorithms.
 
-## Installation
+Pinocchio is open-source, mostly written in C++ with Python bindings, and distributed under LGPL-v3 licence.
+Contributions are welcome.
+
+In this doc, we will find the usual description of the library functionalities, a quick tutorial to catch over the mathematics behind the implementation, a bunch of examples about how to implement classical applications (inverse kinematics, contact dynamics, collision detection, etc) and a set of practical exercices for beginners.
+
+\section OverviewInstall How to install Pinocchio?
+
+Pinocchio is best installed from APT packaging on Ubuntu 14.04 and 16.04, from our repository.
+Every released is validated in the main Linux distribution and OSX, for which installation from source should be straightforward.
 
 The full installation procedure can be found on the Github Pages of the project:
 http://stack-of-tasks.github.io/pinocchio/download.html.
 
-## Simplest example with compilation command
+\section OverviewSimple Simplest example with compilation command
+
+We start by a simple program to compute the robot inverse dynamics. It is given in both C++ and Python version.
+
+<table class="manual">
+<tr><th>C++ version</th><th>Python version</th></tr>
+<tr><td>
+\include overview-simple.cpp
+</td>
+<td>
+\include overview-simple.py
+</td></tr></table>
+
+\subsection OverviewSimpleCompile Compiling and running your program
+
+You would compile the C++ version by including Pinocchio and Eigen header directories.
+
+\code g++ -I /usr/include/eigen -I /path/to/pinocchio/include sample-rnea.cpp -L /path/to/pinocchio/lib -lpinocchio -o sample-rnea \endcode
```

Comment:
The path '/usr/include/eigen' is not correct.

Reply:
Since the installation path may vary for different users, I suggest to write something generic like /path/to/eigen

Reply:
Or simpler than that, you should use pkg-config for that:
`pkg-config --cflags eigen3`

Reply:
No, keep it simple. We should put either /usr/include/eigen3 or /path/to/eigen/include . I think the first solution is clear enough and is easier to use in 90% of the cases, but would agree to use the second one if you believe it is misleading. We should avoid more complex procedure like pkg-config.

Although we might refer the reader to the example dir, where the example will finally be compiled (compilation of example dir is not yet set up).

Reply:
Ok to keep it simple. I support /path/to/eigen/include. Additionally, maybe we can give /usr/include/eigen3 as a possible suggestion the text, or hint to pkg-config as a way to retrieve the correct path.